### ISSUE TITLE
Error on load

### ISSUE THREAD

**Issue by Magnum97** [EpicEricEE/ShopChest#218]
Shopchest seems to work fine but on reload shops are cleared. I tried MySQL but it does not create the table. Here are the errors I get on restart: http://www.hatebin.com/vbpcdqoqna

I removed my config directory and restarted fresh. These are the error I get: http://www.hatebin.com/qgwsieinsr

Versions:
CraftBukkit version git-Spigot-2cf50f0-2b93d83 (MC: 1.12.2) (Implementing API version 1.12.2-R0.1-SNAPSHOT)

ShopChest: 1.12.3

**Comment by EpicEricEE** [comment]
There are two issues:
- First, the update checker, which has already been fixed a while ago (not released yet)
- ShopChest is unable to get a connection to your MySQL server for some reason. You can enable the debug log in the config, reload or restart the server and if you get the error again, check the debug log for more info (or let me know if you don't know what's wrong)

**Comment by Magnum97** [comment]
I set update check to false but still get an error - not a big deal there. I have debug enabled but can not find a debug file. Should that be in plugins/ShopChest ? I checked my SQL database and it has made a connected and created tables, but I have a strange thing now when I do /shop it says Unknown command. Same from console. If I change back to SQLite /shop works again. Also while MySQL is set it will not create debug.txt but it will when set to SQLite.

**Comment by EpicEricEE** [comment]
> I set update check to false

I assume you're using the [latest dev build](https://ci.codemc.org/job/EpicEricEE/job/ShopChest/lastBuild/).

The database type should not prevent the debug log file to be created. Do you get any messages in the console that hint to something being wrong?

**Comment by Magnum97** [comment]
> As that is not possible with the currently released version, I assume you're using the latest dev build.

Haha! I read about adding that to config I think on another issue so I did. I noticed it did not seem to have effect I guess that why ^_^  Also I was not running the version linked, but I will be in a few minutes. 
As far as the database type and no debug... I am clueless as to why. I realize that makes no sense and the only erros are what I linked above. No matter- going on with dev version now to see how we fair. Thank you so much for the help. My users want this in their life so much. 
I'll close this as I no longer have errors on loading.
